commands/acr-build.ts

```diff
@@ -0,0 +1,99 @@
+import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry';
+import { QuickBuildRequest } from "azure-arm-containerregistry/lib/models";
+import { Registry } from 'azure-arm-containerregistry/lib/models';
+import { ResourceManagementClient } from 'azure-arm-resource';
+import { BlobService, createBlobServiceWithSas } from "azure-storage";
+import * as vscode from "vscode";
+import { ImageNode } from "../explorer/models/imageNode";
+import { AzureUtilityManager } from "../utils/azureUtilityManager";
+import { acquireResourceGroup, acquireSubscription, quickPickACRRegistry } from './utils/quick-pick-azure';
+let tar = require('tar');
+let fs = require('fs');
+let os = require('os');
+let url = require('url');
+
+export async function queueBuild(context?: ImageNode): Promise<void> {
+
+    console.log("Obtaining Subscription and Client");
```

Comment:
Throughout the codebase, I haven't seen any console.logs either. They won't show up for the user, so maybe consider removing log statements?